lib/gulp-html5-lint.js

```diff
@@ -5,14 +5,35 @@ var _ = require('ls-lodash'),
     htmlLint = require('html5-lint'),
     gutil = require('gulp-util'),
     PluginError = gutil.PluginError,
-    pluginName = require('../package').name;
+    pluginName = require('../package').name,
+    request = require('sync-request');
 
 module.exports = function gulpHtml5Lint(options) {
     var linterOpts = _.safeMerge(options),
-        errorOutputList = [];
+        errorOutputList = [],
+        req,
+        apiAvailable;
+
+    try {
+        req = request('GET', linterOpts.website || 'https://html5.validator.nu', {
+            timeout: 3000
```

Comment:
It's unclear what this value should be given that html5-lint calls `request` with no `timeout` option.  Testing from different geographies, the response time is sometimes greater than 3 seconds, so we may want to up this to 10 seconds or so for folks with high latency.